packages/rocketchat-ui-sidenav/side-nav/livechatInquiryItem.js

```diff
@@ -0,0 +1,65 @@
+Template.livechatInquiryItem.helpers({
+    alert: function() {
+        if (FlowRouter.getParam('_id') !== this.rid || !document.hasFocus()) {
+            return this.alert;
+        }
+    },
+    unread: function() {
+        return 1;
+    },
+    userStatus: function() {
+        return 'status-' + (Session.get('user_' + this.name + '_status') || 'offline');
+    },
+    name: function() {
+        return this.name;
+    },
+    roomIcon: function() {
+        return RocketChat.roomTypes.getIcon(this.t);
+    },
+    active: function() {
+        if (Session.get('openedRoom') === this.rid) {
+            return 'active';
+        }
+    },
+    route: function() {
+        return RocketChat.roomTypes.getRouteLink(this.t, this);
+    }
+});
+
+Template.livechatInquiryItem.rendered = function() {
+    if (!((FlowRouter.getParam('_id') != null) && FlowRouter.getParam('_id') === this.data.rid) && !this.data.ls && this.data.alert === true) {
+        return KonchatNotification.newRoom(this.data.rid);
+    }
+};
+
+Template.livechatInquiryItem.events({
+    'click .open-room': function(e) {
+        e.stopPropagation();
+        e.preventDefault();
+
+        var inquiry = this;
+
+        swal({
+            title: t('Livechat_Take_Confirm'),
+            text: t('Message') + ": " + inquiry.message,
+            showCancelButton: true,
+            confirmButtonColor: '#3085d6',
+            cancelButtonColor: '#d33',
+            confirmButtonText: 'Take it!'
+        }, function(isConfirm) {
+            if (isConfirm) {
+                // make server method call
+                Meteor.call('livechat:takeInquiry', inquiry, Meteor.user(), function(error, result) {
+                    if (!error) {
+                        swal("success!")
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2756771202/issues/source?bid=3199634&fileBranchId=3348698#l54)